_data/chains/epi155-710420.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Tiltyard Subnet",
+  "chain": "TILTYARD",
+  "rpc": [
+    "https://subnets.avax.network/tiltyard/mainnet/rpc"
+  ],
```

Looks like the CI failed due to formatting of this line.  I think it needs to be formatted like this?

```suggestion
  "rpc": ["https://subnets.avax.network/tiltyard/mainnet/rpc"],
```